webui/react/src/hooks/useModal/Model/useModalModelCreate.tsx

```diff
@@ -175,21 +181,19 @@ const useModalModelCreate = (): ModalHooks => {
           </>
         ) :
           <p className={css.expandDetails} onClick={openDetails}>Add More Details...</p>}
-      </div>
+      </Form>
     );
-  }, [ updateModelName, updateModelDescription, updateMetadata, updateTags, openDetails ]);
+  }, [ form, updateMetadata, updateTags, openDetails ]);
 
   const handleCancel = useCallback(() => modalClose(), [ modalClose ]);
 
   const getModalProps = useCallback((state: ModalState): Partial<ModalFuncProps> => {
-    const { modelName } = state;
     return {
       className: css.base,
       closable: true,
       content: getModalContent(state),
       icon: null,
       maskClosable: true,
-      okButtonProps: { disabled: modelName === '' },
```

Comment:
Somewhat this caused highlight issue, so i removed it and replace it with `Form` to validate